wallet/src/descriptor/mod.rs

```diff
@@ -961,4 +978,36 @@ mod test {
 
         Ok(())
     }
+
+    #[test]
+    fn test_multipath_descriptor_validation() {
```

Comment:
Doesn't this test duplicate the test logic in `test_check_wallet_descriptor()` ? If so it should be removed.